posts/discriminative-vs-generative/index.qmd

```diff
@@ -0,0 +1,595 @@
+---
+title: "Generative vs. discriminative models: what's the difference really?"
+subtitle: "Thinking through the differences between generative and discriminative models."
+date: "2022/04/03"
+image: "params.png"
```

Comment:
thinking one of the factor graphs might be a nicer display image that this. thoughts?

Reply:
What do you mean?

Reply:
"what image do we want on the list of blog posts?"

Reply:
chose to use one of the FGs. let me know if you would rather another image.

Reply:
Did you want to change the title? Also, we should capitalise the D in discriminative.